packages/api/src/controllers/webhook.ts

```diff
@@ -241,28 +241,57 @@ app.post(
   async (req, res) => {
     const { id } = req.params;
     const webhook = await db.webhook.get(id);
-    if (!webhook) {
-      return res.status(404).json({ errors: ["not found"] });
+    if (!webhook || webhook.deleted) {
+      return res.status(404).json({ errors: ["webhook not found or deleted"] });
```

Comment:
For the user, it's just a not found!
```suggestion
      return res.status(404).json({ errors: ["webhook not found"] });
```

Reply:
I added deleted because it's never a user call but an admin one! But yeah, could remove it!